lib/b_main.ml

```diff
@@ -985,6 +991,17 @@ let create ?shortcuts ?(connections = []) ?on_user_event windows =
     mouse_alive = false;
     on_user_event }
 
+let get_monitor_refresh_rate board =
+    Option.bind Layout.(window_opt board.windows_house) @@ fun win ->
```

Comment:
`window_opt board.windows_house` will always return `None` because it is not associated to any physical window.
The problem is that SDL (and bogue) allows to work with several windows. A workaround would be to select the first element of the `board.windows`